assignment.js

```diff
@@ -113,6 +126,60 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+let playerAScore = 0
+let playerBScore = 0
+let tieScore = 0
+const choices = ["rock","paper","scissors"]
+
+for (let i = 0; i < 30; i++) {
+
+    var playerAChoice = choices[Math.floor(Math.random()*choices.length)]
+    var playerBChoice = choices[Math.floor(Math.random()*choices.length)]
+
+
+      if (playerAChoice === playerBChoice ) {
+          console.log("tie")
+          tieScore++
+      }
+
+      if ((playerAChoice === "rock") && (playerBChoice === "scissors")) {
+          console.log("playerA wins")
+          playerAScore++
+      }
+
+      if ((playerAChoice === "scissors") && (playerBChoice === "rock")) {
+          console.log("playerB wins")
+          playerBScore++
+      }
+
+      if ((playerAChoice === "paper") && (playerBChoice === "rock")) {
+          console.log("playerA wins")
+          playerAScore++
+      }
+
+      if ((playerAChoice === "rock") && (playerBChoice === "paper")) {
+          console.log("playerB wins")
+          playerBScore++
+      }
+
+      if ((playerAChoice === "paper") && (playerBChoice === "scissors")) {
+          console.log("playerB wins")
+          playerBScore++
+      }
+
+      if ((playerAChoice === "scissors") && (playerBChoice === "paper")) {
+          console.log("playerA wins")
+          playerAScore++
+        }
+      console.log("==================")
+}
+
+console.log(`playerA Score: ${playerAScore}`)
+console.log(`playerB Score: ${playerBScore}`)
+console.log(`# of Tie Scores: ${tieScore}`)
+
+
+
 // Add your code above this line
```

Comment:
@thetripler Good work here using the Math.random method to generate a random value to pick a game choice from the choice array. The conditionals are good for evaluating the outcomes in your solution. One suggestion instead of using a loop for the game you can wrap it up in a function that you can call at the beginning of the game. Currently, the game is looping 30 times with the same choices and if you wrap it up in a function you can avoid that.
Take a look at this approach [here](https://github.com/jsr20190826/hw-01-js-basics-control-flow-loops-solution/blob/a810fe3f581584e5015519a3508b078d38c4c8c9/assignment.js#L129)